### ISSUE TITLE
SBOM powerup

### ISSUE THREAD

**Issue by sparrell** [sFractal-Podii/quizquadaminos#8]
after #9 If SBOM question is answered correctly, instead of returning to unpaused game, it returns to paused game with one block 'blinking'. Future story will let user select different block but for this story, it is fixed. User typing SBOM will cause the selected block to disappear. Remaining blocks will 'fall' accordingly (that part may need to be separate story) and game continues.

**Comment by sparrell** [comment]
this story replaced by others
